docs/en/operations/system-tables/session_log.md

```diff
@@ -0,0 +1,77 @@
+# system.session_log {#system_tables-session_log}
+
+Contains information about all successful and failed login and logout events.
+
+Columns:
+
+-   `type` ([Enum8](../../sql-reference/data-types/enum.md)) — Login/logout result. Possible values:
+    -   `LoginFailure` — Login error.
+    -   `LoginSuccess` — Successful login.
+    -   `Logout` — Logout from the system.
+-   `auth_id` ([UUID](../../sql-reference/data-types/uuid.md)) — Session id.
+-   `session_id` ([String](../../sql-reference/data-types/string.md)) — Session name.
```

Comment:
Wrong. `auth_id` is not a `session id`, they're different things! Session ID is passed by client via HTTP interface. Please write that and also put here a link to the explanation in https://clickhouse.com/docs/en/interfaces/http/

`auth_id` is authentication ID, it's an UUID which is generated automatically each time a user logins.

Reply:
> Wrong. `auth_id` is not the same as `session_id`, they're different things! Session ID is passed by client via HTTP interface. Please write that and also put here a link to the explanation in https://clickhouse.com/docs/en/interfaces/http/
> 
> `auth_id` is authentication ID, it's an UUID which is generated automatically each time a user logins.

@vitlibar Thank you. I fixed.